docker-for-mac/release-notes.md

```diff
@@ -13,35 +13,65 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
-## Docker Desktop 3.3.3
-2021-05-06
+## Docker Desktop 3.4.0
+2021-06-07
 
 [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
 [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
 
-## Upgrades
+### New
```

Comment:
Should we add a note on Volume Management and Compose canary roll out here?

Reply:
Ah yes, indeed. I’ll get some content from the blog post draft.

Reply:
Yes. And what about the update notifications changes? (cc @dieucao)

Reply:
Yes, totally forgot about that. Good point. I believe we should mention the change to update notification, but @dieucao's call :)

Reply:
I grabbed selected sentences from the blog post. I’m not sure the joyful tone is right for release notes, though. 😁